services/121-service/src/user/approver/entities/approver.entity.ts

```diff
@@ -0,0 +1,18 @@
+import { Column, Entity, JoinColumn, OneToOne, Relation } from 'typeorm';
+
+import { Base121Entity } from '@121-service/src/base.entity';
+import { ProgramAidworkerAssignmentEntity } from '@121-service/src/programs/entities/program-aidworker.entity';
+
+// ##TODO: name OK? program-approver? program-payment-approver?
```

Comment:
I think this is not needed in this case, more of a gut feel that this is implicit